src/Headers.ts

```diff
@@ -147,4 +155,15 @@ export default class HeadersPolyfill {
       }
     }
   }
+
+  /**
+   * Returns an array containing the values
+   * of all Set-Cookie headers associated
+   * with a response
+   */
+  getSetCookie(): string[] {
+    const setCookieHeader =
```

Comment:
So do we expect to return falsy values? I think with header values the intention is to always return an empty string if the value is falsy. Looks like we should adjust this to `X || ''`, not `??`.